addons/spreadsheet/static/src/global_filters/helpers.js

```diff
@@ -91,78 +91,69 @@ export function checkFilterFieldMatching(fieldMatchings) {
     return CommandResult.Success;
 }
 
-/**
- * Get a date domain relative to the current date.
- * The domain will span the amount of time specified in rangeType and end the day before the current day.
- *
- *
- * @param {Object} now current time, as luxon time
- * @param {number} offset offset to add to the date
- * @param {import("@spreadsheet").RelativePeriod} rangeType
- * @param {string} fieldName
- * @param {"date" | "datetime"} fieldType
- *
- * @returns {Domain|undefined}
- */
-export function getRelativeDateDomain(now, offset, rangeType, fieldName, fieldType) {
+export function getRelativeDateFromTo(now, offset, rangeType) {
```

Comment:
I had to keep these two functions outside of the plugin, to keep the tests the same